gateway/metrics/add_metrics_test.go

```diff
@@ -55,6 +56,34 @@ func Test_PrometheusMetrics_MixedInto_Services(t *testing.T) {
 	}
 }
 
+func Test_MetricHandler_ForwardsErrors(t *testing.T) {
```

Comment:
Could you add one more test to show the positive case, if we have that already, just let me know and I'll go ahead and approve.